components/file_attachment_list/index.js

```diff
@@ -19,7 +19,7 @@ function makeMapStateToProps() {
         const fileInfos = selectFilesForPost(state, postId);
 
         let fileCount = 0;
-        if (ownProps.post.metadata) {
+        if (ownProps.post.metadata && ownProps.post.metadata.files) {
```

Comment:
minor nit, I suppose we could simplify it further using:
```suggestion
        if (ownProps.post.metadata?.files) {
```
Not a blocking change request, looks good in combination with the redux PR otherwise!

Reply:
Thanks for the reminder re: the `?.` operator, @DSchalla. I /think/ these are still only available in the mobile repository -- need to add 'plugin-proposal-optional-chaining' to the webapp. Going to queue a dev meeting item to move that along.